README.md

```diff
@@ -54,30 +56,33 @@ $apiKey = 'myKey';
 $imageClient = Factory::getImageClient($organization, $apiKey);
 ```
 
-There is an optional fourth parameter to specify the base URL of the Rokka API. This usually does not need to be adjusted.
+There is an optional parameter to specify the base URL of the Rokka API. This usually does not need to be adjusted.
 
 ### Options for clients
 
-You can add an options array as 1st or 3rd parameter to Factory::getUserClient` or `Factory::getImageClient`.
+You can add an options array as last parameter to Factory::getUserClient` or `Factory::getImageClient`.
 It takes the following format:
 
-```
+```php
 [ 
    Factory::API_BASE_URL => 'https://some-other-api.rokka.io',
    Factory::RENDER_BASE_URL => 'https://myimages.example.com', // you you want/have another render base url
    Factory::PROXY => 'http://proxy:8888', // if you need to use a proxy
    Factory::GUZZLE_OPTIONS => ['verify' => false] // any guzzle option you need/want
 ]
-
 ```
 
 ## Usage
 
-See the [official documentation](https://rokka.io/documentation) on how to use the Rokka API.
+Read the [Getting Started](https://rokka.io/documentation/guides/get-started.html) guide of the [rokka.io documentation](https://rokka.io/documentation) to learn about the basic concepts of rokka.
+
+The image client and user client provide the operations described in the API Reference section of the rokka documentation.
 
-## API Docs
+For example, to upload an image, you would do:
 
-See als the [API Docs](https://rokka.io/client-php-api/master/) for further information.
+```php
+
```

Comment:
Is the example missing here? :)

Reply:
hm. this was some old changes i found on my disk... i will try to figure out what i intended to do here

Reply:
guess i wanted to add an example. i removed this bit and add back the link to the api docs to be able to merge this as the rest of the changes would be useful.